workers/r_dependencies.R

```diff
@@ -0,0 +1,209 @@
+install.packages('ff', repos='https://cloud.r-project.org/')
+install.packages('XML', repos='https://cloud.r-project.org/')
+install.packages('RCurl', repos='https://cloud.r-project.org/')
+install.packages('RSQLite', repos='https://cloud.r-project.org/')
+
+source('https://bioconductor.org/biocLite.R')
```

Comment:
This opens another big security hole into our image. We are placing lots of trust in the folks at `bioconductor.org` to keep tight control (and preferably an audit log) over changes to that `biocLite.R` file... in the wrong hands, that script can turn the installed base of bioconductor into a giant botnet.